src/crsAdapter/CetsAdapter.js

```diff
@@ -538,12 +538,14 @@ class CetsAdapter {
         service.travellers.forEach((serviceTraveller, index) => {
             const traveller = this.helper.traveller.normalizeTraveller(serviceTraveller);
 
+            const travellerName = traveller.name.split(' ');
             xml.Fap.push({
                 [CONFIG.builderOptions.attrkey]: {
                     ID: index + 1,
                 },
                 PersonType: traveller.salutation,
-                Name: traveller.name,
+                Name: travellerName[1],
+                FirstName: travellerName[0],
```

Comment:
maybe it is better to do something like:
```
Name: travellerName.pop(),
FirstName: travellerName.join(' '),
```

so you get definitely the complete name. maybe not in the correct fields - but still.